tools/Environments/DevHome.Environments/ViewModels/LandingPageViewModel.cs

```diff
@@ -157,22 +250,61 @@ public void SearchHandler(string query)
         };
     }
 
+    /// <summary>
+    /// Updates the view model to filter the compute systems according to the provider.
+    /// </summary>
+    [RelayCommand]
+    public void ProviderHandler()
+    {
+        var currentProvider = Providers[SelectedProviderIndex];
+        ComputeSystemsView.Filter = system =>
+        {
+            if (currentProvider == _stringResource.GetLocalized("AllProviders"))

```

Comment:
We probably should make sure `All providers` stays at the top of the list regardless of localization

Reply:
Yeah, it is made sure of in earlier code.

Reply:
Oh, just realized you're not sorting them in the combo box, should they be sorted by name?